hack/tests/e2e-helm.sh

```diff
@@ -28,7 +29,7 @@ remove_operator() {
     kubectl delete --ignore-not-found=true clusterrolebinding nginx-operator-system-metrics-reader
     kubectl delete --ignore-not-found=true --namespace=${test_namespace} -f "$OPERATORDIR/config/samples/helm.example_v1alpha1_nginx.yaml"
     kubectl delete --ignore-not-found=true namespace ${test_namespace}
-    kubectl delete --ignore-not-found=true namespace ${operator_namespace}
+    make undeploy
```

Comment:
I do not think that it will work. See that we need to use $OPERATORDIR to delete the samples here.

Reply:
I saw that you make worked 👍